src/docfx/build/legacy/files/LegacyPage.cs

```diff
@@ -44,17 +36,9 @@ internal static class LegacyPage
 
                 var themesRelativePathToOutputRoot = "_themes/";
 
-                if (!string.IsNullOrEmpty(doc.RedirectionUrl))
-                {
-                    rawMetadata = LegacyMetadata.GenerateLegacyRedirectionRawMetadata(docset, pageModel);
```

Comment:
remove `LegacyMetadata.GenerateLegacyRedirectionRawMetadata()`, not in use any more

Reply:
These lines are covered in LegacyTOC, I think they are kept here accidentally during refactor.